src/molecules/Toggle/ToggleExample.jsx

```diff
@@ -0,0 +1,20 @@
+
+const ToggleExample = (props) => {
+  const {
+    className,
+    children,
+    ...rest
+  } = props;
+
+  return(
+    <div>
+      {props.toggle ? 
+        <p className="Toggle__on">Toggle switch "ON"</p> :
+        <p className="Toggle__off">Toggle switch "OFF"</p>
+      }     
+      </div>
+  )
+  
```

Comment:
Can you add some PropTypes validation here?

Reply:
@Brookeruu - could you define the PropTypes for the `onChange` prop as well? After you have done so, please feel free to merge / delete this branch.

If you want me to show you had to do that, please ping me and I'll show you :)